def/core.js

```diff
@@ -360,12 +360,12 @@ def("Comment")
 
 // Block comment. The .type really should be BlockComment rather than
 // Block, but that's what we're stuck with for now.
-def("Block")
+def("CommentBlock")
     .bases("Comment")
     .build("value", /*optional:*/ "leading", "trailing");
```

Comment:
Oh, yeah, maybe `CommentBlock` and `CommentLine` should go in `def/babel.js`.

The order of evaluation doesn't matter, but it's nice to keep it clear that these types are Babel/Esprima-specific.

Also, you can delete the `//` comments above the definitions where I complain about the Esprima type names :)